src/modm/architecture/interface/can_message.hpp

```diff
@@ -23,7 +23,7 @@ namespace modm::can
 /// @ingroup modm_architecture_can
 struct Message
 {
-	Message(const uint32_t& inIdentifier = 0, uint8_t inLength = 0) :
+	inline Message(const uint32_t& inIdentifier = 0, uint8_t inLength = 0) :
```

Comment:
`uint32_t inIdentifier` should be passed by value according to current best practices (c++ core guidelines, etc.).